Modules/DataModel/Sources/PocketCastsDataModel/Public/DataManager.swift

```diff
@@ -417,6 +417,10 @@ public class DataManager {
         return episodeManager.findBy(uuid: uuid, dbQueue: dbQueue)
     }
 
+    public func findEpisodeCount(podcastId: Int64) async -> Int {
```

Comment:
Question: this is all sync. So basically, we're doing an `async` wrap on a sync operation.

Couldn't we just do something like?

```swift
Task {
    syncMethod()
}
```

The end result should be the same.